### PR TITLE
Use Scalafmt Maven plugin

### PR DESCRIPTION
**Pull request by pstutz**

### CODE REVIEW

**Comment by pstutz** [comment]
There's also an IntelliJ plugin that allows to format on save.

**Comment by pstutz** [comment]
The Maven plugin is rather slow. Might make more sense for all of us to use the IntelliJ plugin and enable `format on save`.

File: caps-core/src/main/scala/org/opencypher/caps/CAPS.scala [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/**
+/*
```
Comment (Mats-SX) [inline caps-core/src/main/scala/org/opencypher/caps/CAPS.scala]: I'm not sure, but I think there should be double asterisks for the license comment.

**Comment by boggle** [comment]
I have mixed feelings on this; I occasionally like to layout code for readability not sure this is achievable with this. Is there a comment for locally disabling scalafmt?

File: caps-core/src/main/scala/org/opencypher/caps/CAPS.scala (hunk 1 above)
Comment (thobe) [reply caps-core/src/main/scala/org/opencypher/caps/CAPS.scala]: No. No there should not.

Comment (pstutz) [reply caps-core/src/main/scala/org/opencypher/caps/CAPS.scala]: Both IntelliJ formatting and scalafmt disagreed with our license plugin, so I changed its settings to something that the auto-formatters wouldn't touch.

**Comment by pstutz** [comment]
Yes, there's an escape hatch: https://github.com/scalameta/scalafmt/issues/91 
```
// format: OFF
val identityMatrix = Array(1, 0, 0
                                            0, 1, 0
                                            0, 0, 1)
// format: ON
```

**Comment by pstutz** [comment]
Decision at standup: find Scalafmt configuration that causes minimal changes to the current style.

**Pull request closed without merging**